snippets/xfs-filesystem-snippet.adoc

```diff
@@ -0,0 +1,13 @@
+// Text snippet included in the following modules:
+//
+// * modules/oadp-1-3-backing-csi-snapshots.adoc
+// * modules/
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+If you format the volume by using XFS filesystem and the volume is at 100% capacity, the backup fails with the following error: `Error: relabel failed /var/lib/kubelet/pods/3acf.../pvc-68..c/mount/data-xfs-103: no space left on device`  
```

Comment:
Is it always this error? I am concerned the path might be for the customer only

Reply:
Thanks Andy. I reworded the note to include an example of the error message.